include/seqan3/io/sequence_file/format_genbank.hpp

```diff
@@ -0,0 +1,252 @@
+                                            std::string{"ORIGIN"})))
+                {
+                        std::ranges::copy(stream_view | view::take_line_or_throw
+                                                	  | view::char_to<value_type_t<id_type>>,
+                                     	  std::back_inserter(id));
+                        std::ranges::copy(std::string{"\n"}, std::back_inserter(id));
+                }
+            }
+            else
+            {
+				// ID
+                detail::consume(stream_view | view::take_until(!is_blank));
+                // read id
+                std::ranges::copy(stream_view | view::take_until_or_throw(is_cntrl || is_blank)
+                                              | view::char_to<value_type_t<id_type>>,
+                                  std::back_inserter(id));
+                detail::consume(stream_view | view::take_line_or_throw);
+            }
+        }
+
+        // Jump to sequence
+        while (!(std::ranges::equal(stream_view | view::take_until_or_throw(is_cntrl || is_blank),
+                                    std::string{"ORIGIN"}) | options.embl_genbank_complete_header))
+        {
+            detail::consume(stream_view | view::take_until(is_cntrl));
+            ++stream_it;
+        }
+
+        // Sequence
+        detail::consume(stream_view | view::take_line_or_throw);
+        auto constexpr is_end = is_char<'/'> ;
+        if constexpr (!detail::decays_to_ignore_v<seq_type>)
+        {
+            auto constexpr is_legal_alph = is_in_alphabet<seq_legal_alph_type>;
+            std::ranges::copy(stream_view | ranges::view::remove_if(is_space || is_digit)
+                                          | view::take_until_or_throw_and_consume(is_end) // consume //
+                                          | std::view::transform([is_legal_alph] (char const c) // enforce legal alphabet
+                                         {
+                                             if (!is_legal_alph(c))
+                                             {
+                                                 throw parse_error{std::string{"Encountered an unexpected letter: "} +
+                                                                   is_legal_alph.msg.string() +
+                                                                   " evaluated to false on " +
+                                                                   detail::make_printable(c)};
+                                             }
+                                             return c;
+                                         })
+                                          | view::char_to<value_type_t<seq_type>>,    // convert to actual target alphabet
+                                            std::back_inserter(sequence));
+        }
+        else
+        {
+            detail::consume(stream_view | view::take_until_and_consume(is_end));
```

Comment:
use the throw version here just like in the other branch above (`take_until_or_throw_and_consume`)